AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -332,6 +332,7 @@ forbes.com#@#.AD-Rotate
 !
 @@||fireman.club/wp-content/themes/fireclub/ads/banner.gif$domain=fireman.club
 @@||static.criteo.net/images/pixel.gif$domain=gmx.net
+||absorbingprison.com^$script,domain=velonews.com
```

Comment:
$third-party, and add to Tracking Protection

Reply:
and remove this rule from annoyances?
I think we should leave it in Annoyances too